src/components/AlertBox.tsx

```
@@ -0,0 +1,117 @@
+    border: '1px solid var(--dark20)',
+    borderRadius: '0.5rem',
+    padding: '0.5rem',
+    color: 'white',
+    width: '100%',
+  }
+  return (
+    <div style={{ padding: '2px', width: '100%' }}>
+      <div style={style}>
+        <FlexRow gap='0.5rem'>
+          <AlertBox icon={<MegaphoneIcon animated />}>
+            <div dangerouslySetInnerHTML={{ __html: html }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)